projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.5-a.3 - 2021-12-14
+### Enhancements
+- JITM: allow sidebar JITMs marked as dismissible to be dismissed.
```

Comment:
I don't think "JITM" is really helpful for most users. I would move this to "Other".